README.md

```diff
@@ -1,8 +1,111 @@
+
+## For more information
+On our [website](https://magmo.com) you will find links to our whitepapers and contact information. Whether you simply want to try
+out our apps, or get involved more deeply we would love to hear your thoughts. Deployed versions of our games may be accessed with these links:
+
+* [Rock Paper Scissors](https://demo.magmo.com) (RPS)
+* [Tic Tac Toe](https://ttt.magmo.com) (TTT)
+
+## Setting up development environment and running a game application
+You will need `yarn` installed (see [here](https://yarnpkg.com/lang/en/docs/install/) for instructions). After cloning the code, 
+1. In the top directory, run `yarn install`.
+2. In the top directory, run `npx lerna bootstrap`.
+3. Add `.env` files to the root directory of the `wallet` package, and to the root directory of the relevant app directory. Refer to `.env.example` which includes an example of the required variables.
+
+4. Start [ganache](https://truffleframework.com/ganache) by running `yarn ganache:start` .
+5. (In a new tab) Run the wallet via `yarn start` in the `wallet` package directory
+6. (In a new tab) Run a game (either RPS or TTT) via `yarn start` in the relevant package directory.
+7. Add [MetaMask](https://metamask.io/) to your browser, and point it to `localhost:3001` to view the application. You will need to import one of the seed accounts (below) into metamask to have funds to transact.
+    1. Open the metamask browser extension
+    2. Click on the account icon (circle in the top right)
+    3. Select "Import"
+    4. Paste in one of the secret keys below.
+
+You may visit the app in two different browsers in order to play against yourself. We use [Redux DevTools](https://github.com/reduxjs/redux-devtools) to develop and test our apps.
+
+
+#### Seed accounts
+Please refer to [this file](https://github.com/magmo/devtools/blob/master/utils/startGanache.js).
+
+#### To run storybook
+
+We use [Storybook](https://storybook.js.org/) to view our react components during development. You can start Storybook by running:
+```
+yarn storybook
+```
+This will fire up the Storybook panel inside a browser.
+
+
+#### To build:
+
+1. Update your  `TARGET_NETWORK` in all relevant `.env` files to a named network from `truffle.js` (default is `ropsten`)
+2. Build the application:
+    ```
+    yarn run build
+    ```
+
+#### To deploy smart contracts
+
+1. Add test eth to your account for the deployment using an eth faucet: https://faucet.ropsten.be/ or https://faucet.metamask.io.
+2. Set `TARGET_NETWORK` in your `.env` file to the network you want to deploy to: either `'development'`, `'ropsten'`, `'kovan'` or `'rinkeby'`.
+3. Deploy the contracts to the network:
+    ```
+    yarn deployContracts
+    ``` 
+
+#### Running Tests specific to a certain app
+From the relevant subdirectory...
+* To run application tests in watch mode:
+
+`yarn test:app`
+
+* To run smart contract tests:
+
+`yarn test:contracts`
```

Comment:
there's also `yarn test:integration` in some packages, right?

Reply:
- In wallet 
```
"test:integration": "npx ganache-then-jest -c ./config/jest/jest.integration.config.js --runInBand"
```
- In rps 
```
"test:contracts": "npx ganache-then-jest -c ./config/jest/jest.integration.config.js --runInBand"
```
- in ttt  
```
"test:contracts": "node scripts/test_truffle.js",
```

Let's make these consistent?